README.md

```diff
@@ -92,6 +92,7 @@ application.on(application.launchEvent, (args) => {
 #### ⚠️ Note: Xcode 10 & iOS 12+ ⚠️
 
 If developing using Xcode 10 and targeting iOS devices running iOS 12 or higher, the "Access WiFi Information" capability is required in order to discover and connect to Cast devices.
+The plugin comes with an `app.entitlements` which will add this capability the the workspace, however, you must also `Add the Access WiFi information feature to your App ID` . 
```

Comment:
"the the", can you fix the spelling mistake.

Reply:
done